include/trieste/xoroshiro.h

```diff
@@ -27,6 +27,8 @@ namespace xoroshiro
       }
 
     public:
+      typedef RESULT result_type;
```

Comment:
The code base tends to use `using`.  I don't think I have seen a `typedef` anywhere in the code.
```suggestion
      using result_type = RESULT;
```

Reply:
That makes perfect sense. Sorry for being lazy and not attentive :|

Reply:
No need to apologise.  There are no coding docs ;-)

It is just I have to think to read `typedef` because it is backwards.